bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -73,9 +73,12 @@
   private InetSocketAddress address;
   private final ExecutorService serverHelperExecutor;
   private final ExecutorService storageHelperExecutor;
+  private boolean backPressureEnabled;
 
   private byte[] authToken;
 
+  private static final boolean BACK_PRESSURE_ENABLED = !Boolean.getBoolean("BUFFERSERVER.BACK_PRESSURE_DISABLE");
```

Comment:
Since it is a global server level setting currently, I prefer to leave it in the Server as opposed to putting it in the data list.